conf/lsf.conf

```diff
@@ -39,11 +37,11 @@ backend {
             -q ${lsf_queue} \
             -n ${cpu} \
             ${"-g " + lsf_job_group} \
-            -R "select[singularity] rusage[mem=${memory_mb}] span[hosts=${hosts}]" \
+            -R "rusage[mem=${memory_mb}] span[hosts=${hosts}]" \
```

Comment:
`~{}` syntax is preferred to `${}`. For two reasons, 1) `${}` isn't supported at all in `<<<...>>>` task definitions, 2) is a personal opinion, I think using `$` can make it unclear on a quick glance whether it's a Bash variable or a WDL variable. I suggest changing all instances of `${}` to `~{}`.

Reply:
Agreed. I changed the syntax to use ~{} instead of ${}.